projects/fourier/test/fourier/services/test/tuist/unit_test.rb

```diff
@@ -17,9 +17,6 @@ def test_call
             Utilities::System
               .expects(:tuist)
               .with("test")
-            Utilities::System
-              .expects(:system)
-              .with("swift", "test")
```

Comment:
This is the internal unit test for fourier. What this does is verify that when you call `fourier test tuist unit`, fourier will call `tuist dependencies fetch`, `tuist test`, and `swift test`. Removing these 3 lines will make the unit test no longer expect `swift test` to be called, but won't actually change the fact that it is called. Thus the test will now fail.